llvm/lib/SYCLLowerIR/LowerESIMD.cpp

```diff
@@ -1244,6 +1244,11 @@ PreservedAnalyses SYCLLowerESIMDPass::run(Module &M,
 
 size_t SYCLLowerESIMDPass::runOnFunction(Function &F,
                                          SmallPtrSet<Type *, 4> &GVTS) {
+  // Add 'alwaysinline' attribute to every function called from ESIMD kernel
```

Comment:
I suggest to change the comment to explain why this is needed.

Reply:
Done.